example/README.md

```diff
@@ -0,0 +1,70 @@
+In this folder, we can see four main packages in different sub-folders:
+- register_certificate
+- register_contract
+- execute_contract
+- validate_ledger
+
+They demonstrate how to use Scalar DL Go Client SDK to send basic operation requests to Scalar DL networks.
+
+To build all of them
+```
+cd <sub-folder>
+go build
+```
+
+## Local Scalar DL Network
+We need a local Auditor-enabled Scalar DL network before we run these examples.
+
+Reference [scalardl-samples
+](https://github.com/scalar-labs/scalardl-samples) to start up local Scalar DL networks.
+
+## client.properties
+The Scalar DL Client Properties example can be found in [client.properties](client.properties).
+
+This properties example is configured to connect to a local and auditor-enabled Scalar DL network.
+
+### register_certificate
```

Comment:
Should we have how to use them instead of how to run built commands, like the node SDK?
https://github.com/scalar-labs/scalardl-node-client-sdk

Reply:
I also revised the README for a brief introduction in https://github.com/scalar-labs/scalardl-go-client-sdk/pull/20/commits/74c9d369ce2f7bc384e758d13fc2cdf3584f53fd